ui/app/serializers/oidc/provider.js

```diff
@@ -2,4 +2,15 @@ import ApplicationSerializer from '../application';
 
 export default class OidcProviderSerializer extends ApplicationSerializer {
   primaryKey = 'name';
+
+  normalizeItems(payload) {
+    if (payload.data) {
+      if (payload.data?.keys && Array.isArray(payload.data.keys)) {
+        return payload.data.keys.map((key) => ({ name: key, ...payload.data.key_info[key] }));
```

Comment:
nice!

Reply:
Could you add a small comment about what information it is you're extracting here? e.g. something that a future developer could read to better understand why we have this in the serializer. My thought is that in the future it won't be very clear why we have this. We might know why now, but someone not involved with the project would have to dig through the response to understand this addition to the serializer.